qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -18,6 +26,12 @@ You can download an example sample information file and prep information file fr
 Sample information file
 -----------------------
 
+Similar to a `QIIME metadata file, <http://qiime.org/documentation/file_formats.html#metadata-mapping-files>`__
```

Comment:
> However, it is conceptually different, since the QIIME metadata file contains information from the Sample Information file and the Prep information file. For example, in the QIIME metadata file there is the BarcodeSequence column. That would be conceptually incorrect to add in the Sample Information file. 

Well said. 

This is exactly the distinction I struggled to grasp when I first tried to submit to qiita. I'll see how best I can write this up.